recipes/aws-cdi-sdk/all/conandata.yml

```diff
@@ -0,0 +1,22 @@
+sources:
+  "2.2.0":
+    - url: https://github.com/aws/aws-cdi-sdk/archive/refs/tags/v2.2.0.tar.gz
+      sha256: 4ee109b7fa2f683b4b0cd03028d0946857f09f34da4e89fe69a5c297cdaeb689
+    - url: https://raw.githubusercontent.com/aws/aws-sdk-cpp/1.8.130/cmake/compiler_settings.cmake
```

Comment:
why does `2.2.0` downloads some sources from `1.8.130`? it looks suspicious...
are they missing from the distribution? if so, why not take them from 2.2.0 tag?